frame/multisig/src/benchmarking.rs

```diff
@@ -237,6 +188,8 @@ benchmarks! {
 		assert_eq!(multisig.approvals.len(), 2);
 	}
 
+	/// TODO: in the documentation it's said that the last approval should be `as_multi` which
```

Comment:
@gavofyork @kianenigma this is confusing to me, according to [this doc](https://github.com/paritytech/substrate/blob/7bacd730ff42afe9d6c5039a51e74bd461dac5ef/frame/multisig/src/lib.rs#L389) we should be using `as_multi` for final approvals. Yet the benchmark exists and it fails every since the Call storage has been removed.

Reply:
@kianenigma I have checked the logic now and it all makes sense. Since when we are using `approve_as_multi` without a `Call` storage - there's nothing to call. We can remove this bench and the docs are now correct, which they were not before.